src/scripts/quick_reblog.js

```diff
@@ -218,6 +221,33 @@ const updateQuickTags = (changes, areaName) => {
   }
 };
 
+/**
+ * adapted from https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#converting_a_digest_to_a_hex_string
+ *
+ * @param {string} data - String to hash
+ * @returns {Promise<string>} Hexadecimal string of a unique hash of the input
+ */
+const sha256 = async (data) => {
+  const msgUint8 = new TextEncoder().encode(data);
+  const hashBuffer = await crypto.subtle.digest('SHA-256', msgUint8);
+  const hashArray = Array.from(new Uint8Array(hashBuffer));
+  const hashHex = hashArray.map(b => b.toString(16).padStart(2, '0')).join('');
+  return hashHex;
+};
+
+const updateRememberedBlog = async event => {
```

Comment:
In most cases I find it good practice to use `currentTarget` over `target`, as it ensures you're getting the data from the element you're attaching the event listener to (assuming you're attaching the function directly, as we are here).

```suggestion
const updateRememberedBlog = async ({ currentTarget: { value: selectedBlog } }) => {
```